src/applications/ezr/tests/e2e/ezr-emergency-contacts.cypress.spec.js

```diff
@@ -0,0 +1,107 @@
+      statusCode: 200,
+      body: mockPrefill,
+    }).as('mockSip');
+  });
+
+  it('should add emergency contacts', () => {
+    cy.visit(manifest.rootUrl);
+    cy.wait(['@mockUser', '@mockFeatures', '@mockEnrollmentStatus']);
+
+    advanceToEmergencyContacts();
+
+    selectYesNoWebComponent('view:isEmergencyContactsEnabled', true);
+    goToNextPage(
+      '/update-benefits-information-form-10-10ezr/veteran-information/emergency-contacts/0/contact',
+    );
+
+    let contact = testData.veteranContacts[0];
+    // ec 1 basic info
+    cy.get(`[name="root_fullName_first"]`)
+      .first()
+      .type(contact.fullName.first);
+    cy.get(`[name="root_fullName_last"]`)
+      .first()
+      .type(contact.fullName.last);
+    fillTextWebComponent('primaryPhone', contact.primaryPhone);
+    selectDropdownWebComponent('relationship', contact.relationship);
+    selectYesNoWebComponent('view:hasEmergencyContactAddress', true);
+
+    // ec 1 address
+    goToNextPage(
+      '/update-benefits-information-form-10-10ezr/veteran-information/emergency-contacts/0/contact-address',
+    );
+    selectDropdownWebComponent('address_country', contact.address.country);
+    fillTextWebComponent('address_street', contact.address.street);
+    fillTextWebComponent('address_city', contact.address.city);
+    selectDropdownWebComponent('address_state', contact.address.state);
+    fillTextWebComponent('address_postalCode', contact.address.postalCode);
+
+    cy.tabToElementAndPressSpace('.usa-button-primary');
+
+    cy.findByText(/Review your emergency contacts/i).should('exist');
+    cy.findByText(`${contact.fullName.first} ${contact.fullName.last}`).should(
+      'exist',
+    );
+    cy.findByText(contact.primaryPhone).should('exist');
+
+    selectYesNoWebComponent('view:isEmergencyContactsEnabled', true);
+    cy.tabToElementAndPressSpace('.usa-button-primary');
+
+    contact = testData.veteranContacts[1];
+    // ec 2 basic info
+    cy.get(`[name="root_fullName_first"]`)
+      .first()
+      .type(contact.fullName.first);
+    cy.get(`[name="root_fullName_last"]`)
+      .first()
+      .type(contact.fullName.last);
+    fillTextWebComponent('primaryPhone', contact.primaryPhone);
+    selectDropdownWebComponent('relationship', contact.relationship);
+    selectYesNoWebComponent('view:hasEmergencyContactAddress', true);
+
+    // ec 2 address
+    goToNextPage(
+      '/update-benefits-information-form-10-10ezr/veteran-information/emergency-contacts/1/contact-address',
+    );
+    selectDropdownWebComponent('address_country', contact.address.country);
+    fillTextWebComponent('address_street', contact.address.street);
+    fillTextWebComponent('address_city', contact.address.city);
+    selectDropdownWebComponent('address_state', contact.address.state);
+    fillTextWebComponent('address_postalCode', contact.address.postalCode);
+
+    cy.tabToElementAndPressSpace('.usa-button-primary');
+
+    // review page
+    cy.get('va-alert').should(
```

Comment:
I'm not a fan of checking content in e2e tests. This functionality is unit tested and unnecessary for a Cypress spec. We should be checking behavior, not content.